config/initializers/omniauth.rb

```diff
@@ -13,3 +13,4 @@
 end
 
 OmniAuth.config.allowed_request_methods = [:post, :get]
+OmniAuth.config.silence_get_warning = true
```

Comment:
@asideofcode-dev is this needed because Omniauth v2.0.0 only allows post requests? But we are also allowing get on line 15?

Reply:
Yup, that's it. `GET` is considered unsafe. Technically we can migrate to using `POST` but the signin/signup flow is currently using redirects in various places which don't play well with `GET` e.g. `AuthServicesController#new`